<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IND040101141" failures="2" tests="3" name="Test(failed)" time="22.809" errors="0" timestamp="2025-04-29T12:19:50 IST">
  <testcase classname="testCases.TC002_ValidateLoggingIntoTheApplicationUsingValidCredentials" name="validLoginTest" time="2.053"/>
  <testcase classname="testCases.TC003_LoginDDT" name="validate_ddt" time="2.53">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.validate_ddt(TC003_LoginDDT.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validate_ddt -->
  <testcase classname="testCases.TC003_LoginDDT" name="validate_ddt" time="2.438">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.validate_ddt(TC003_LoginDDT.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validate_ddt -->
</testsuite> <!-- Test(failed) -->
